.github/workflows/ci.yml

```diff
@@ -16,11 +16,8 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: [ubuntu-latest]
+        os: [ubuntu-latest, windows-latest, macos-latest]
```

Comment:
Yes, this is great!

Reply:
we intentionally moved away from that in a previous PR and our tests sped up significantly (GitHub bottlenecks open source projects from running too many actions at once, so iirc it essentially doubled our test suite wait time 2->5 minutes). I can try to find that PR if it would be useful context.

I don't think anything valuable is gained by this change either, in practice. In the lifespan of that old setup, I can't think of a time where this additional testing has caught an issue of Windows or Mac. 

Overall I'm -1 on this change. If there's still interest in discussing, I would at the very least recommend breaking it out into a separate PR so that it's not blocking the larger macOS testing work (which I am +1 on)

Reply:
That’s fair. I’ll add a comment in that case describing the reason for the split Unix vs Windows/macOS tests.

Reply:
To Fred’s point, yes there is a max cap on individual actions that can be run at once, so running more versions of Windows/mac can mean that 1 fewer PR can run at a time. How much that affects this project depends on the cadence of PRs, CI runs, and pushes. I guess I observed it at a low-volume time and Fred is speaking to high-volume times.

Also as an advocate for fast tests (I put work into Mocha parallel tests), I don’t mind over-indexing for fast tests in most cases.

Reply:
Just to add some real numbers to this, I found the docs on limits: https://docs.github.com/en/actions/learn-github-actions/usage-limits-billing-and-administration

**20 concurrent jobs / 5 concurrent macOS jobs** = only 1 PR test suite could fully run at a time (10 total jobs, 3 macOS jobs).